config/checkstyle/checkstyle.xml

```diff
@@ -212,7 +212,7 @@
       <property name="option" value="NL"/>
       <property name="tokens"
                 value="BAND, BOR, BSR, BXOR, DIV, EQUAL, GE, GT, LAND, LE, LITERAL_INSTANCEOF, LOR,
-                    LT, MINUS, MOD, NOT_EQUAL, PLUS, QUESTION, SL, SR, STAR, METHOD_REF "/>
+                    LT, MINUS, MOD, NOT_EQUAL, QUESTION, SL, SR, STAR, METHOD_REF "/>
```

Comment:
I'll make a note on this for other reviewers' benefit.

This is a change I made as TL. It's low-priority, doesn't affect the application logic and is within the domain of TLs, not devs.

Standard practice would be to have a separate ticket for this, so please keep in mind this is a bad practice I am adopting here and DON'T do it yourselves. I packed it in this ticket to see how reception of this change would have been.